app/views/api_docs/pages/release_notes.md

```diff
@@ -3,3 +3,8 @@
 Initial release of the API.
 
 For a log of pre-release changes, [see the alpha release notes](/api-docs/alpha-release-notes).
+
+#### 7th January 2020
```

Comment:
Perhaps at this add the top to have reverse chronological order like the alpha release notes.

Reply:
Yes